app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command):
+    split_file = command.split(" ")
+    if "/" in split_file[-1]:
+        path = split_file[-1].split("/")
+        os.makedirs("/".join(path[:-1]))
```

Comment:
Use `os.path.join` to concatenate parts of the path. 
In that case, your app will be cross-platform and will 
use either `/` or `\` depending on OS